cmd/agent/agent.go

```diff
@@ -280,3 +284,31 @@ func stringSliceAddToMap(sl []string, m map[string]string) error {
 	}
 	return nil
 }
+
+func readAgentId(agentIdConfigPath string) int64 {
+	const defaultAgentIdValue = int64(-1)
+
+	rawAgentId, fileErr := os.ReadFile(agentIdConfigPath)
+	if fileErr != nil {
```

Comment:
log err

Reply:
Should we make it debug as it will otherwise popup each time at the first start for normal runs.